index.jsx

```diff
@@ -196,7 +229,8 @@ export const updateState = (event, previousState) => {
   const numHearts = (percentage / 100) * MAX_HEARTS
   const fullHearts = Math.floor(numHearts)
   const remainder = numHearts - fullHearts
-  
+  const magic = event.output.match(/\d+\.\d+%\sidle/)[0].replace('%', '').replace('idle', '')/100
```

Comment:
How does this work, a few lines earlier I `parseInt` on event.output, expecting output to just be a number.

Reply:
I don't know regex well enough to explain exactly but the parseInt above doesn't modify event.out so the data string data about the CPU will still be there.

Reply:
got it, i see, I think i will tweak this code later to make it a little more clear.